packages/card/stories/RadioCard.stories.tsx

```diff
@@ -0,0 +1,66 @@
+import type { StoryFn, StoryObj } from '@storybook/react';
+
+import { RadioCard } from '../src/RadioCard';
+
+export default {
+  component: RadioCard,
+  title: 'Components/Card/RadioCard',
+  description: 'A radio button with a label and optional image and subtext.',
+  decorators: [
+    (Story: StoryFn) => (
+      <div style={{ display: 'flex', width: '24rem' }}>
+        <Story />
+      </div>
+    ),
+  ],
+  parameters: {
+    status: {
+      type: import.meta.env.STORYBOOK_PACKAGE_STATUS__RADIOCARD,
+    },
+  },
+};
+
+type Story = StoryObj<typeof RadioCard>;
+
+const commonProps = {
+  label: 'Label',
```

Comment:
Believe `id` is also required to ensure the label maps to the radio properly. Could you add one here so that the a11y addon in Storybook has no violations?